core/trietxn.go

```diff
@@ -0,0 +1,68 @@
+package core
+
+import (
+	"github.com/bits-and-blooms/bitset"
+	"github.com/dgraph-io/badger/v3"
+)
+
+// TrieTxn is a database transaction on a trie.
+type TrieTxn struct {
```

Comment:
Final comment, can we please rename it to `TrieBadgerTxn`

Reply:
Done